devdocs-browser.el

```diff
@@ -684,7 +684,7 @@ Result is a plist metadata, with an extra :index field at the beginning."
                      devdocs-browser-doc-base-url)))
           ;; write data to files
           (dolist (kv (seq-partition data 2))
-            (when-let* ((name (substring (symbol-name (car kv)) 1))
+            (when-let* ((name (convert-standard-filename (substring (symbol-name (car kv)) 1)))
                         (value (cadr kv))
                         ;; prepent "./" to fix paths starting with literal "~" (e.g. deno)
                         (path (expand-file-name (concat "./" name ".html") data-dir)))
```

Comment:
Should we apply `convert-standard-filename` here (around `expand-file-name`)? The document of `convert-standard-filename` mentions that "FILENAME should be an absolute file name"

Reply:
You are right.